README.md

```diff
@@ -5,61 +5,63 @@ Accelerated Container Image is an open-source implementation of paper ["DADI: Bl
-It can be used for container acceleration by supporting fetching image data on-demand without downloading and unpacking the whole image before a container running. With OverlayBD image format, we can cold start a container instantly.
+At the heart of the acceleration is overlaybd, which provides a merged view of a sequence of block-based layers as an iSCSI block device.
+It can be used for container acceleration by supporting fetching image data on-demand without downloading and unpacking the whole image before a container running. With overlaybd image format, we can cold start a container instantly.
 
 The key features are:
 
 * **High Performance**
 
-    It's a block-device-based storage of OCI image, which has much lower complexity than filesystem-based implementations. For example, cross-layer hardlink and non-copy commands like chown are very complex for filesystem-based image without copying up, but is natively supported by OverlayBD. OverlayBD outperforms filesystem-based solutions in performance. Evaluation data is stated in [DADI paper](https://www.usenix.org/conference/atc20/presentation/li-huiba).
+    It's a block-device-based storage of OCI image, which has much lower complexity than filesystem-based implementations. For example, cross-layer hardlink and non-copy commands like chown are very complex for filesystem-based image without copying up, but is natively supported by overlaybd. Overlaybd outperforms filesystem-based solutions in performance. Evaluation data is stated in [DADI paper](https://www.usenix.org/conference/atc20/presentation/li-huiba).
 
 * **High Reliability**
 
-    OverlayBD outputs block devices through iSCSI protocol, which is widely used and supported in most operation systems. OverlayBD backing-store can recover from failures or crashes.
+    Overlaybd outputs block devices through iSCSI protocol, which is widely used and supported in most operation systems. Overlaybd backing-store can recover from failures or crashes.
 
 * **Native Support for Writable**
 
-    OverlayBD can be used as writable/container layer. The end-users can build their OverlayBD images naturally without conversion.
+    Overlaybd can be used as writable/container layer. The end-users can build their overlaybd images naturally without conversion.
+
+## Components
+
+* [overlaybd](https://github.com/alibaba/overlaybd)
+
+    Overlaybd provides a merged view of block-based layer sequence as a third-party backing-store of tgt, which is an user space iSCSI target framework.
+
+* overlaybd-snapshotter
+
+    It's just what this repo focuses on: a [containerd](https://containerd.io/) snapshotter plugin for overlaybd image. This snapshotter is compatible for OCI image, as well as overlayfs snapshotter.
+
+    We also provide a modified CLI tool(ctr) to facilitate image download and custom conversion, between traditional OCI tarball format and overlaybd's obd format.
```

Comment:
I think 'image download' is ambiguous, it actually does not download the image.
'overlaybd's obd format', just like 'overlaybd format' or 'overlaybd-format' is ok.

Reply:
download -> pull

overlaybd's obd format  -> overlaybd format